app/Locale/sv_SE/translations.php

```diff
@@ -730,52 +730,52 @@
     'End date' => 'Slutdatum',
     // 'Users overview' => '',
     'Members' => 'Medlemmar',
-    // 'Shared project' => '',
+    'Shared project' => '',
```

Comment:
Please, pay attention to the unit tests. They are failing. The translation cannot be empty.

https://travis-ci.org/kanboard/kanboard/jobs/586615106